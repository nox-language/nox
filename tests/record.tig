let type Point = {
        x: int,
        y: int
    }
    var point := Point {
        x = 42,
        y = 24
    }
in (
    printi(point.x);
    printi(point.y);
    point.y := 100;
    printi(point.x);
    printi(point.y);
    point.x := 200;
    printi(point.x);
    printi(point.y)
)
end
