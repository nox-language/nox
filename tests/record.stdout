42
24
42
100
200
100
